tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,83 @@
-
-      //Need to call sendTx in order to make it always with value < 1000, otherwise it fails
-      await rskClient.rsk.sendTx({
-        from: addresses.rsk,
-        to: rsk.getBridgeAddress(),
-        value: WEIS_TO_RELEASE,
-        gasPrice: TO_BRIDGE_GAS_PRICE
-      }, rskClient.evm.mine);
-
-      //Call release after sendTx so requests are released. Goes one by one
-      await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
-
-      await wait(500);
-      
-      var afterFirstReleaseFedBalance = Number((await btcClient.getAddressBalance(federationAddress))[federationAddress]);
-      expect(afterFirstReleaseFedBalance).to.be.at.most(INITIAL_BTC_BALANCE / 2 + initialFedBalance);
-
-      //Need to call sendTx in order to make it always with value < 1000, otherwise it fails
-      await rskClient.rsk.sendTx({
-        from: addresses.rsk,
-        to: rsk.getBridgeAddress(),
-        value: WEIS_TO_RELEASE,
-        gasPrice: TO_BRIDGE_GAS_PRICE
-      }, rskClient.evm.mine);
-
-      //Call release after sendTx so requests are released. Goes one by one
-      await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
-      
-      var finalFedBalance = Number((await btcClient.getAddressBalance(federationAddress))[federationAddress]);
-      expect(finalFedBalance).to.be.at.most(initialFedBalance + bitcoin.btcToSatoshis(1));
-    } 
-    catch (err) {
-      throw new CustomError('Transfer BTC to RBTC failure', err);
-    }
+    const btcAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+    await whitelistingAssertions.assertAddLimitedLockWhitelistAddress(rskTxHelper, btcAddressInfo.address, PEGIN_VALUE_IN_SATOSHI);
+    await rskUtils.mineAndSync(rskTxHelpers);
+    
+    const recipientRskAddressInfo = getDerivedRSKAddressInformation(btcAddressInfo.privateKey, btcTxHelper.btcConfig.network);
+    await rskTxHelper.importAccount(recipientRskAddressInfo.privateKey);
+    const unlocked = await rskTxHelper.unlockAccount(recipientRskAddressInfo.address);
+    expect(unlocked, 'Account was not unlocked').to.be.true;
+
+    const latestActiveForkName = getLatestActiveForkName();
+    const bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+    const federationAddress = await bridge.methods.getFederationAddress().call();
+
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
+
+    await btcTxHelper.fundAddress(btcAddressInfo.address, PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+    
+    const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, PEGIN_VALUE_IN_BTC);
+    await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+    const recipientRskAddressBalanceAfterPegin = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+    expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(Number(3 * MINIMUM_PEGOUT_VALUE_IN_WEIS));
```

Comment:
```suggestion
    expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(Number(PEGIN_VALUE_IN_BTC));
```

Reply:
hmm... the `recipientRskAddressBalanceAfterPegin` returns balance in weis, so that is why i'm asserting weis
`const recipientRskAddressBalanceAfterPegin = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));`

Reply:
`   expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(Number(PEGIN_VALUE_IN_WEISS));`